Library/Homebrew/dependency_collector.rb

```diff
@@ -65,6 +65,8 @@ def fetch(spec)
   def cache_key(spec)
     if spec.is_a?(Resource) && spec.download_strategy <= CurlDownloadStrategy
       File.extname(spec.url)
+    elsif spec.is_a?(UsesFromMacOSDependency)
+      "#{spec.name}-#{spec.bounds}"
```

Comment:
There's probably a better way to do this, but this seems to work.

Reply:
I think a better way is to fix `def hash` and `def ==` in `UsesFromMacOSDependency`

Reply:
Sorry, enabled auto-merge before I saw your comment @Bo98.